lib/cocoapods/installer/analyzer/target_inspector.rb

```diff
@@ -0,0 +1,196 @@
+module Pod
+  class Installer
+    class Analyzer
+      class TargetInspector
+        include Config::Mixin
```

Comment:
since this is only used for `config.installation_root`, please pass that in as an arg to the initializer?